src/ml-engine/main_app/infrastructure/defi_llama.py

```
@@ -1,30 +1,117 @@
+    mu: Optional[float]
+    poolMeta: Optional[str]
+    tvlUsd: Optional[int]
+    volumeUsd1d: Optional[float]
+    volumeUsd7d: Optional[float]
+    sigma: Optional[float]
+    underlyingTokens: List[str] = field(default_factory=list)
+    rewardTokens: List[str] = field(default_factory=list)
+
+
+def get_pool_summary_data() -> Dict[str, List[PoolData]]:
     url = f"https://yields.llama.fi/pools"
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)